topics/metagenomics/tutorials/metatranscriptomics/content.md

```diff
@@ -1068,8 +1068,8 @@ Although gene families and pathways, and their abundance may be related to a spe
 > ### {% icon hands_on %} Hands-on: Combine taxonomic and functional information
 >
 > 1. {% tool [Combine MetaPhlAn2 and HUMAnN2 outputs](toolshed.g2.bx.psu.edu/repos/bebatut/combine_metaphlan2_humann2/combine_metaphlan2_humann2/0.1.0) %}  with the following parameters:
->   - {% icon param-file %} *"Input file corresponding to MetaPhlAN2 output"*: `Cut predicted taxon relative abundances table`
->   - {% icon param-file %} *"Input file corresponding HUMAnN2 output"*: `Normalized gene families`
+>   - {% icon param-file %} *"Input file corresponding to MetaPhlAN output"*: `Cut predicted taxon relative abundances table`
+>   - {% icon param-file %} *"Input file corresponding HUMAnN output"*: `Normalized gene families`
 >   - *"Type of characteristics in HUMAnN2 file"*: `Gene families`
```

Comment:
>   - *"Type of characteristics in HUMAnN file"*: `Gene families`

Reply:
Thank you so much @subinamehta :)
I have just updated it in a new commit